### ISSUE TITLE
Syntax Highlighting doesn't work because of Uncaught SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by munchkindev** [CarlosTenorio/ng-docx#27]
I have followed the manual on how to implement ng-docx into an Angular 10 project, but when I launch the route the following error appears and the syntax highlighting doesn't work because of the missing imports:
`Uncaught SyntaxError: Cannot use import statement outside a module`
The problem at hand appears to be this code in node_modules/ng-docx/assets/js/ng-docx.js:
```javascript
import 'node_modules/marked/lib/marked.js';
import 'node_modules/prismjs/prism.js';
import 'node_modules/prismjs/components/prism-typescript.min.js';
import 'node_modules/prismjs/plugins/line-numbers/prism-line-numbers.js';

```
How do you fix this?

**Comment by munchkindev** [comment]
Also if I try to use the Search bar I get the following error when clicking on it and nothing happens it seems:
```
ERROR TypeError: Cannot read property 'getBoundingClientRect' of undefined
```

**Comment by CarlosTenorio** [comment]
1. **Uncaught SyntaxError: Cannot use import statement outside a module**
Here you have an [example with Angular 10](https://stackblitz.com/edit/ng-docx-demo-ng10). Can you try import directly on you angular.json

```
"scripts": [
    "node_modules/marked/lib/marked.js",
    "node_modules/prismjs/prism.js",
    "node_modules/prismjs/components/prism-typescript.min.js",
    "node_modules/prismjs/plugins/line-numbers/prism-line-numbers.js"
]
```

2. **ERROR TypeError: Cannot read property 'getBoundingClientRect' of undefined**
I can reproduce on my [stackblitz fro Angular 10](https://stackblitz.com/edit/ng-docx-demo-ng10). I will create a BUG to solve it. Thanks

**Comment by CarlosTenorio** [comment]
Fixed on version 0.4.2 🚀

**Comment by CarlosTenorio** [comment]
Fixed on version 0.4.2 🚀
